selfdrive/ui/ui.cc

```diff
@@ -264,7 +264,7 @@ void UIState::update() {
   emit uiUpdate(*this);
 }
 
-void UIState::setPrimeType(int type) {
+void UIState::setPrimeType(PrimeType type) {
   if (type != prime_type) {
```

Comment:
I think behaviour has changed since `prime_type` defaults to `PrimeType::NONE` it might not get written to params/event emitted

Reply:
y, this might be a bug.but It looks like the original version also had this issue: the prime_type is set to none in UIState::UIState if PrimeType not existed:
`prime_type = std::atoi(params.get("PrimeType").c_str());`

Reply:
added a new enum value PrimeType::UNKNOWN to fix this issue, what do you think?

Reply:
that seems fine

Reply:
should we add a helper for `hasPrime` in `UIState`?